procedures/katello/branch.md.erb

```diff
@@ -74,10 +73,3 @@
   - [ ] Rename the "Katello Next" release to Katello <%= develop %>.0
   - [ ] Recreate the "Katello Next" release and indicate that it is a placeholder for issues belonging to the next version of Katello
   - [ ] Create the "Katello <%= release %>.1 TODO" custom query using the last one as a template
-
-
-## Release Engineer
-
-- [ ] Run `./tools.rb koji configs/katello/<%= release %>.yaml --confirm` from [tool_belt](https://github.com/theforeman/tool_belt) to create Koji tags
-- [ ] Run `./tools.rb mash-scripts configs/katello/<%= release %>.yaml` from [tool_belt](https://github.com/theforeman/tool_belt) to create Koji mash configs and open PR to tool_belt to commit
-- [ ] Copy mash configs to Koji using <%= rel_eng_script('deploy_mash_configs') %>
```

Comment:
This whole section is now inherently built into the Foreman release process that at the time of branching `rpm/develop` will fork all of the nightly repositories to a versioned repository -- https://github.com/theforeman/theforeman-rel-eng/pull/285/files#diff-d061c99b19d953d05d43c2de1a7199aac667795ed8a2e9c7991cef788118b7f8R74